### ISSUE TITLE
"unsupported state format" message when running tests

### ISSUE THREAD

**Issue by thiagoarrais** [hashicorp/terraform-provider-local#75]
I'm getting failures when running the acceptance tests on a clean clone. Any pointers here?

### Versions

```
$ terraform version
Terraform v1.0.1
on linux_amd64
$ go version
go version go1.16.5 linux/amd64
$ git show HEAD | head -1
commit 0f51dd473181ff71500c16d2009ad953244cec2e
```

### Reproducing

Git clone and run acceptance tests:

```
$ make testacc
```

### Expected Behavior

Acceptance tests run without failure.

### Actual Behavior

Got some failures with an "unsupported state format version" error message.

Here's an instance:

```
=== RUN   TestLocalFileDataSource
=== RUN   TestLocalFileDataSource/0
    data_source_local_file_test.go:33: Step 1/1 error: unsupported state format version: expected "0.1", got "0.2"
    testing_new.go:56: Error retrieving state, there may be dangling resources: unsupported state format version: expected "0.1", got "0.2"
--- FAIL: TestLocalFileDataSource (0.21s)
    --- FAIL: TestLocalFileDataSource/0 (0.21s)
```

**Comment by thiagoarrais** [comment]
It looks like bumping the plugin SDK to 2.7.0 fixes this. I think **dependabot** will get to that sometime?

**Comment by gibsonje** [comment]
Also saw this error in my test provider's test suite out of the blue 4 days ago. The plugin SDK was at v2.6.1. This github issue is the only place I found another instance of this error with regard to a provider's acceptance tests failing. Will try bumping to 2.7.0.

**Comment by thiagoarrais** [comment]
This was haunting [some of my providers](https://github.com/thiagoarrais/terraform-provider-wercel/runs/2941129381?check_suite_focus=true) too, @gibsonje. That was actually how I got to _local_.

As for this issue, I'm closing it since it is fixed in main after 528138ff9 (dependabot really caught it as expected).
